### DISCUSSION TITLE
为什么不训练一个支持中文、英文、公式混合识别的模型

### DISCUSSION THREAD

**Discussion by gk966988** [OleehyO/TexTeller#51] [Q&A] [answered]
目前看到项目中是将文档中的公式检测出来，然后识别，为什么不训练一个模型能够同时识别中文、英文和latex公式呢？

**Comment by OleehyO** [comment, marked as answer]
这个问题[之前](https://github.com/OleehyO/TexTeller/discussions/41)讨论过，总的来说就是端到端识别需要的数据+算力比分阶段识别要多很多。

不过新版的TexTeller（预计在6月10号前发布）已经能同时识别中英文+latex公式了，但也确实用了比较多的资源。

**Comment by gk966988** [comment]
> 这个问题[之前](#27)讨论过，总的来说就是端到端识别需要的数据+算力比分阶段识别要多很多。
> 
> 不过新版的TexTeller（预计在6月10号前发布）已经能同时识别中英文+latex公式了，但也确实用了比较多的资源。

非常棒的工作，非常期待新版本的更新！

**Comment by OleehyO** [comment]
更新了嗷🥳
